pkgs/nix-tools/darwin-rebuild.sh

```diff
@@ -215,7 +215,7 @@ if [ "$action" = switch ] || [ "$action" = activate ] || [ "$action" = rollback
   "$systemConfig/activate-user"
 
   if [ "$USER" != root ]; then
-    sudo -H "$systemConfig/activate"
+    sudo -H --preserve-env=PATH env "$systemConfig/activate"
```

Comment:
```suggestion
    sudo -H --preserve-env=PATH "$systemConfig/activate"
```

Shouldn't be necessary here as `$systemConfig/activate` should be an absolute path